client/apollo/css/src/Form/Radio/CardRadio/CardRadioLF.scss

```
@@ -1,4 +1,4 @@
-@use "RadioCardCommon";
+@use "CardRadioCommon";
 
 .af-radio-card {
```

il faut aussi renommer les class CSS
```suggestion
.af-card-radio {
```

---

C'est traité